packages/astro/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # astro
 
+## 1.0.0
+
+> Astro v1.0 is out! Read the [official announcement post](https://astro.build/blog/astro-1/).
+
+> **Note**: If you need help migrating an existing Astro project to the new Astro v1.0, check out our updated [Migration Guide](https://docs.astro.build/en/migrate/) and [full documentation website](https://docs.astro.build/).
+
+### Features
+
+Astro v1.0 includes a few new features and improvements since our original beta announcement back in April, including:
+
+- [SSR Builds](https://docs.astro.build/en/guides/server-side-rendering/): Server output is now stable and available for production use.
+- [Image Optimization](https://docs.astro.build/en/guides/integrations-guide/image/): New `<Image />` and `<Picture />` components available with [`@astrojs/image`](https://docs.astro.build/en/guides/integrations-guide/image/).
+- [MDX Support](https://docs.astro.build/en/guides/integrations-guide/mdx/): A standard syntax for mixing UI components in Markdown is available with [`@astrojs/mdx`](https://docs.astro.build/en/guides/integrations-guide/mdx/).
+- [Vite 3.0](https://vitejs.dev/blog/announcing-vite3.html): An upgrade to our internal build engine, Vite.
+
+### Breaking Changes
+
+- `Astro.canonicalURL` has been deprecated. Please use [`Astro.url`](https://docs.astro.build/en/reference/api-reference/#astrourl) to construct your own canonical URL.
+- [CSS Specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) is now preserved for scoped styles. See the [migration guide](https://docs.astro.build/en/migrate/#changed-scoped-css-specificity) for more details.
+  > **Warning**: Please visually inspect your site output to make sure everything is styled as expected. If not, find your scoped style and increase the selector specificity manually to match the old behavior.
+- Components and JSX in Markdown has been deprecated. For long-term support you should migrate to the [`@astrojs/mdx`](https://docs.astro.build/en/guides/integrations-guide/mdx/) integration. See the [migration guide](https://docs.astro.build/en/migrate/#deprecated-components-and-jsx-in-markdown) for more details.
+- The previously deprecated `<Markdown>` component has been removed. It is now available as a separate package, [`@astrojs/markdown-component`](https://github.com/withastro/astro/tree/main/packages/markdown/component).
```

Comment:
I'm not sure why these are looking in my preview as if they might be still part of the above warning? (The text is light grey in the file diff view on GitHub) So I guess just double-check any formatting here to make sure the list is rendering properly.

Reply:
You can view it as the final output here! https://github.com/withastro/astro/blob/66453ec6592612cf2b78b525e6db3c62bb7587b3/packages/astro/CHANGELOG.md